guides/deployment/deployment.md

```diff
@@ -57,7 +57,7 @@ Keep in mind that, if you by any chance forget to run the steps above, Phoenix w
 
 ```console
 $ PORT=4001 MIX_ENV=prod mix phx.server
-10:50:18.732 [info] Running MyAppWeb.Endpoint with Cowboy on http://example.com
+10:50:18.732 [info] Running MyAppWeb.Endpoint with Cowboy on https://example.com
```

Comment:
I do not think we should change this URL– the current value matches the default Endpoint configuration from the Phoenix installer.
```suggestion
10:50:18.732 [info] Running MyAppWeb.Endpoint with Cowboy on http://example.com
```